configure.ac

```diff
@@ -9,7 +9,7 @@ AC_GNU_SOURCE
 AC_USE_SYSTEM_EXTENSIONS
 
 AC_ARG_ENABLE(gtk20, 
-              [  --disable-gtk20	  Don't look for GTK+ 2.0 libraries], 
+              [  --disable-gtk20	  Don not look for GTK+ 2.0 libraries], 
```

Comment:
Misspelling.... 'Don not'

Reply:
Fixed. `Don't` breaks geany syntax highlighting..